homeassistant/components/moehlenhoff_alpha2/sensor.py

```diff
@@ -45,10 +48,12 @@ def __init__(
         self.heat_control_id = heat_control_id
         self._attr_unique_id = f"{heat_control_id}:valve_opening"
         heat_control = self.coordinator.data["heat_controls"][heat_control_id]
-        heat_area = self.coordinator.data["heat_areas"][heat_control["_HEATAREA_ID"]]
-        self._attr_name = (
-            f"{heat_area['HEATAREA_NAME']} heat control {heat_control['NR']} valve"
-            " opening"
+        self._attr_device_info = DeviceInfo(
+            identifiers={(DOMAIN, heat_control_id)},
```

Comment:
The identifiers collide with the identifiers of the heat area device.
Can we use `_attr_unique_id` here?
```suggestion
            identifiers={(DOMAIN, self._attr_unique_id)},
```

Reply:
Or use different prefixes for heat area device and heat control device?

Reply:
Can you give an example of all these ids?

Reply:
I think I would prefer not to create these devices at all.
I would add the entities to the heat area device instead and use a `translation_key`.

Reply:
Like yes technically we can do that, but we do have the guideline that a physical device should also be its own device so I'm a bit in the middle

Reply:
Here are two examples from real environments: https://github.com/j-a-n/python-moehlenhoff-alpha2/blob/main/tests/data